components/MaskedTransition/src/private/MDCMaskedTransitionMotionForContext.m

```diff
@@ -16,26 +16,27 @@
 
 #import "MDCMaskedTransitionMotionForContext.h"
 
-MDCMaskedTransitionMotionSpec motionForContext(NSObject<MDMTransitionContext> *context) {
+MDCMaskedTransitionMotionSpecContext
+    MDCMaskedTransitionMotionSpecForContext(NSObject<MDMTransitionContext> *context) {
```

Comment:
Nit: This function doesn't appear to use any NSObject methods.
Can the parameter be:
id<MDMTransitionContext> rather than NSObject<MDMTransitionContext> ?

Reply:
Done.